week-1-practise/level-2/1-tax.js

```diff
@@ -5,8 +5,9 @@
   Sales tax is 20% of the price of the product
 */
 
-function calculateSalesTax() {}
-
+function calculateSalesTax(price) {
+  return price / 5;
```

Comment:
Indeed! I usually see answers that multiply the price by a decimal point, this is interesting :)

Reply:
@martynamaron, thanks